install.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/sh
+
+function install_fail() {
+    echo "Installation failed"
+    exit 1
+}
+
+
+function Help(){
+    echo "Usage: install.sh [option]"
+    echo "Options:"
+    echo "  -h: Show this help message and exit"
+    echo "  -d: Install only dependencies"
+    echo "  -p: Install only python dependencies (including playwright)"
+}
+
+
+
+while getopts ":hydp" option; do
+    case $option in
+        h)
+            Help
+            exit 0;;
+        y)
+            ASSUME_YES=1;;
+        d)
+            DEPS_ONLY=1;;
+        p)
+            PYTHON_ONLY=1;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            Help
+            exit 1;;
+        :)
+            echo "Option -$OPTARG requires an argument." >&2
+            Help
+            exit 1;;
+    esac
+done
+
+
+
+function install_macos(){
+    if [ -z "$(command -v brew)" ]; then
+        NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+        if [[ uname -m == "x86_64" ]]; then
+            echo "export PATH=/usr/local/bin:$PATH" >> ~/.bash_profile && source ~/.bash_profile
+        else
+            echo "export PATH=/opt/homebrew/bin:$PATH" >> ~/.bash_profile && source ~/.bash_profile
+        fi
+    else
+        echo "Homebrew is already installed"
+    fi
+    echo "Installing python"
+    if [ ! command -v python3 &> /dev/null ]; then
```

Comment:
Same staff, command will not work in square parenthesis, u can chose rather use them or not. And change it in all file
```suggestion
    if ! command -v python3 &> /dev/null; then
```
```suggestion
    if [ -z "$(command -v python3)" ]; then
```